ds701_book/04-Linear-Algebra-Refresher.qmd

```diff
@@ -251,53 +251,49 @@ plt.show()
 ## Dot product
 :::
 
-The dot product of two vectors $\mathbf{u}, \mathbf{v}$ can be used to project $\mathbf{u}$ onto $\mathbf{v}$. This is illustrated in @fig-dot-product.
+The dot product of two vectors $\mathbf{u}, \mathbf{v}$ can be used to project $\mathbf{u}$ onto $\mathbf{v}$. 
+
+$$
+\mathrm{proj}_{\mathbf{v}}\mathbf{u} = \frac{\mathbf{u}\cdot\mathbf{v}}{\Vert \mathbf{v} \Vert^2}\mathbf{v}
```

Comment:
I would move the period at the end of the previous sentence and place it at the end of this formula.